### PR TITLE
EIP-1109: Remove call costs for precompiled contracts

### PR DESCRIPTION
**Pull request by jbaylina**

### CODE REVIEW

File: EIPS/eip-1109.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+eip: 1109
+title: Remove CALL costs for precompiled contracts
+author: Jordi Baylina (@jbaylina)
+discussions-to: jordi@baylina.cat
```
Comment (Arachnid) [inline EIPS/eip-1109.md]: Please provide a public URL where people can discuss.

File: EIPS/eip-1109.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+status: Draft
+type: Core
```
Comment (Arachnid) [inline EIPS/eip-1109.md]: You need to supply a category as well as a type; see EIP 0 or EIP-X for guidance.

File: EIPS/eip-1109.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+---
+eip: 1109
+title: Remove CALL costs for precompiled contracts
+author: Jordi Baylina (@jbaylina)
+discussions-to: https://github.com/ethereum/EIPs/pull/1109
```
Comment (Arachnid) [inline EIPS/eip-1109.md]: Please don't use this PR as a persistent discussion venue - either open an issue for discussion, or a thread on Ethereum Magicians.

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Contains new file EIPS/eip-1109.md
 - EIP 1087 requires approval from one of (@arachnid)

File: EIPS/eip-1109.md (hunk 3 above)
Comment (jbaylina) [reply EIPS/eip-1109.md]: Changed!

File: EIPS/eip-1109.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+This EIP is backwards compatible.  Smart contracts that call precompiled contracts will cost less from now on. Smart contracts that relay in a high gas consumption for a specific code are not expected.
+
+## Test Cases
+
+- Normal call to a defined smart contract.
+- Call to undefined smart contract.
+- Call to defined smart contract with a value!=0 on the call.
+- Call to undefined smart contract with a value!=0 on the call.
+- Normal call with remaining gas<700 but gas>[the cost of the call].
+
```
[the remark this answers is not in the document]
Comment (jbaylina) [reply EIPS/eip-1109.md]: Added!

**Pull request merged**